airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -82,11 +83,11 @@ def get_job_group(self, job_id: int, embed: str, include_deleted: bool) -> Dict[
         A job group is a job that is executed from a specific node in a flow.
 
         :param job_id The ID of the job that will be fetched.
-        :type job_id: int
+        :type job_id: int.
         :param embed Comma-separated list of objects to pull in as part of the response.
-        :type embed: string
+        :type embed: string.
```

Comment:
```suggestion
        :type embed: str
```
There's no `string` type

Reply:
My mistake.

Reply:
done